### ISSUE TITLE
Implement baseline preprocessing module

### ISSUE THREAD

**Issue by Markus-Goetz** [helmholtz-analytics/heat#400]
**Related**
Most likely requires #399 to be finished first.

**Feature functionality**
Implement a baseline preprocessing module. Preprocessors are all kind of features scalars (standardizer, normalizer, min-max-scaler, ...). For the first module implementation iteration three preprocessors (the ones previously mentioned) are sufficient. Also expose the non-object oriented module-level functional style calls.

**Additional context**
Sklearn provides a well-designed preprocessor API. One might want to get inspired/adapt their design [https://scikit-learn.org/stable/modules/classes.html#module-sklearn.preprocessing](https://scikit-learn.org/stable/modules/classes.html#module-sklearn.preprocessing)
